django/db/models/functions/text.py

```diff
@@ -174,6 +184,16 @@ def as_oracle(self, compiler, connection):
         return super().as_sql(compiler, connection, function='SUBSTR')
 
 
+class Trim(Transform):
+    function = 'TRIM'
+    lookup_name = 'trim'
+
+
 class Upper(Transform):
     function = 'UPPER'
     lookup_name = 'upper'
+
+
+LStrip = LTrim  # Give LTrim a more Pythonic name.
```

Comment:
I'm not sure these add much value. Users can use `import x as y` if they want aliases?

Reply:
Agreed. I also wasn't convince that aliases add much value.